src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
```

Comment:
This was required by fabpot.io to pass, not sure about it.

Reply:
fabbot has some false-positives, this is one of them :)